implementations.md

```diff
@@ -19,3 +19,25 @@ If you know of any associated projects that are not listed here, please file a p
 ## Testing
 
 * [huawei-openlab/oct](https://github.com/huawei-openlab/oct) - Open Container Testing framework for OCI configuration and runtime
+
+## Hooks
+
+For addressing container issues that are outside the scope of this
```

Comment:
On Thu, Dec 17, 2015 at 10:44:35AM -0800, Alexander Morozov wrote:
> > +For addressing container issues that are outside the scope of this
> 
> I wonder what became with … one sentence per line.

Fixed in 1c301f1 → 3f89557, thanks :).

> I personally hate that rule, but I know that you love it :)

I'm not a fan of that *choice* in particular, but I am a fan of having
a rule of some sort (e.g. I'd also be ok with “Markdown must be
wrapped at 72 characters”, but that's probably harder for folks to
implement consistently).  Bikeshedding about or churning the
repository for re-wrapping Markdown is just a waste of time.